lib/actions/preload.js

```diff
@@ -20,6 +20,27 @@ import * as dockerUtils from '../utils/docker';
 
 const isCurrent = (commit) => commit === 'latest' || commit === 'current';
 
+/** @type {any} */
+const applicationExpandOptions = {
```

Comment:
This is a copy-and-paste from:
https://github.com/balena-io-modules/balena-preload/blob/v10.1.0/lib/preload.js#L778-L796
... in order to avoid `balena-cli` depending on it, given that different SDK versions may need different query data structures.

The `any` type avoids some tsc failures that I struggled to resolve... But it was successfully tested.

Reply:
Looks like the sdk typings do not support the `Array<Dictionary<string>>` notation for `$orderby` :(
Let's do the following though, so that we do get a bit better typings:
```
/** @type {import('balena-sdk').PineExpandFor<import('balena-sdk').Application>} */
const applicationExpandOptions = {
	owns__release: {
		$select: ['id', 'commit', 'end_timestamp', 'composition'],
		/** @type {any} */
		$orderby: [{ end_timestamp: 'desc' }, { id: 'desc' }],
```

Reply:
I'd rather a `// @ts-ignore $reason` or `// @ts-expect-error $reason` for this rather than a cast to `any` with no explanation, with the `ts-ignore`/`ts-expect-error` it's easier to grep for and with the comment it's also possible to know what's necessary to remove the bypass